server/app.ts

```diff
@@ -0,0 +1,58 @@
+import express from 'express';
+import cors from 'cors';
+import createError from 'http-errors';
+import { Application, Request, Response, NextFunction } from 'express';
+import path from 'path';
+
+import { ApiRouter } from './api-router';
+
+/**
+ * Bootrapping of an Application server in express.
+ *
+ * Demonstrates a simple example application using express that a customer
+ * might create for their frontent to communicate with and then to communicate
```

Comment:
```suggestion
 * might create for their front end to communicate with and then to communicate
```
I was curious and found [this] for what it's worth lol(https://css-tricks.com/poll-results-front-end-front-end)

Reply:
Lol interesting